packages/proxyIdentity/src/offerableIdentityOperator.ts

```diff
@@ -77,6 +83,10 @@ export class OfferableIdenitytOperator extends Operator {
     validity?: number
   ): Promise<BigNumber> {
     const identity = addressOf(did);
+    const identityOwner = await this.identityOwner(did);
+    if (!(identityOwner === this.identity.address)) {
+      throw new OnlyOwnerAllowed(identityOwner, await this.getOwnerAddress());
```

Comment:
The error message is `Signer ${signer} is not allowed to update DID identity ${identity}`, so shouldn't `did` be the second param?
```suggestion
      throw new OnlyOwnerAllowed(identityOwner, did);
```

Reply:
Do you mean that order of params in message should be the same as order in error call? I put identity on the first place in constructor, because `identity` seems to me primary in this event. So that when you write `OnlyOwnerAllowed` the first what you expect is whose owner? 
Do you think preserving order has some benefits